test/src/module/storage/azureTest.c

```diff
@@ -1026,6 +1026,113 @@ testRun(void)
 
                 TEST_RESULT_VOID(storagePathRemoveP(storage, STRDEF("/path"), .recurse = true), "remove");
 
+                // -----------------------------------------------------------------------------------------------------------------
+                TEST_TITLE("switch to time limited");
+
+                hrnServerScriptClose(service);
+
+                hrnCfgArgRawZ(argList, cfgOptPgPath, "/pg1");
+                hrnCfgArgRawZ(argList, cfgOptRepo, "1");
+                hrnCfgArgRawZ(argList, cfgOptLimitTime, "2024-08-04 02:54:09+00");
```

Comment:
This is static because we're mocking the output from storage server?

Reply:
Exactly. Makes the tests far simpler.